common/spec/dependabot/notices_spec.rb

```diff
@@ -204,33 +142,4 @@ def initialize(name:, version:, deprecated_versions: [], unsupported_versions: [
         })
     end
   end
-
-  describe ".generate_pm_unsupported_notice" do
```

Comment:
Not used generate pm unsupported notice method is removed after finding out that we don't need to create notice. Instead we are using general error flow to raise unsupported error.